safe/impact_functions/volcanic/volcano_polygon_building/impact_function.py

```diff
@@ -46,6 +46,9 @@ def __init__(self):
         self.volcano_names = tr('Not specified in data')
         self._target_field = 'Hazard'
 
+        # From BuildingExposureReportMixin
+        self.building_report_threshold = 25
```

Comment:
In case of a mixin, python will look for the first ```__init__``` method from the left to right. For instance, in our code : 
```
class VolcanoPolygonBuildingFunction(
        ClassifiedVHClassifiedVE,
        BuildingExposureReportMixin):
    """Impact Function for Volcano Point on Building."""
```
Only the ```__init__``` in ```ClassifiedVHClassifiedVE``` will be executed. That's why I had to duplicate that part of code in every IF. But it's true that this value is going to be overwrite. What should I do ? :)